gradle.properties

```diff
@@ -1,4 +1,5 @@
 javaVersion=11
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
Is this really necessary? 
This is equal to default value on my machine (this can be obtained by `java -XX:+PrintFlagsFinal -version | grep MaxHeapSize`), so, in my case this does nothing, aside that better not to have gradle specific arguments configuration checked in (we already did, and then we came back)

Reply:
Removed.

Reply:
It that causes problems in future only in the CI, a cli parameter on the CI script parameter could be added